assembly/mode7-demo.asm

```diff
@@ -38,6 +38,11 @@ loop3:	lda (2),y
 	dex
 	bne loop3
 
+	+vset vreg_lay2 | AUTO_INC_1
```

Comment:
Layer two's address space is accessed only once.  Therefore, `| AUTO_INC_1` isn't needed.